doc/expressionLanguage.md

```diff
@@ -176,152 +237,7 @@ following to CB:
-the value that is being sent.
-
-Currently, the expression parser does not support JSON Arrays and JSON document. A new issue has been created to address
-this aspect https://github.com/telefonicaid/iotagent-node-lib/issues/568. For the rest of types the workflow will be the
-following:
-
-1. Variables will be cast to String no matter the expression type (see [comments above](#types) regarding this)
-2. The expression will be applied
-3. The output type will be cast again to the original attribute type.
-
--   If attribute type is `Number` and the value is an `Integer`, then the value is casted to integer (JSON number)
--   If attribute type is `Number` and the value is a `Float`, then the value is casted to float (JSON number)
--   If attribute type is `Boolean` then the value is cast to boolean (JSON boolean). In order to do this conversion,
-    only `true` or `1` are cast to true.
--   If attribute type is `None` then the value is cast to `null` (JSON null)
-
-> **Note**. All the operations and castings described above are not performed using JELX, because the user has full 
-> control on the final value of the attributes, so there is no need of adding a layer of autocast that would interfere
-> and makes things more complicated.
-
-E.g.: if a device with the following provisioning information is provisioned in the IoT Agent:
-
-```json
-{
-    "name": "status",
-    "type": "Boolean",
-    "expression": "${@status *  20}"
-}
-```
-
-and a measurement with the following values arrive to the IoT Agent:
-
-```
-status: true
-```
-
-1. The expression `*` is a multiplication, so the expression type makes `status` to be casted to Number. The cast of
-   `true` to number is 1.
-2. Expression is evaluated, resulting in 20
-3. 20 is cast to `20` since Expression Plugin always use String as Expression type.
-4. The attribute type is `Boolean` so the result is casted to Boolean before sending it to CB. The cast of `20` to
-   boolean is false (only `true` or `1` are cast to true).
-
-More examples of this workflow are presented below for the different types of attributes supported in NGSI v2 and the
-two possible types of expressions: Integer (arithmetic operations) or Strings.
-
--   `pressure` with value 52 (integer)
--   `consumption` with value 0.44 (float)
--   `active` with value `null` (None type)
-
-| Expression              | Expected outcome | Format  |
-| :---------------------- | :--------------- | ------- |
-| `${@pressure * 20}`     | `1040`           | Integer |
-| `${trim(@pressure)}`    | `52`             | Integer |
-| `${@consumption * 20}`  | `8.8`            | Float   |
-| `${trim(@consumption)}` | `0.44`           | Float   |
-| `${@pressure * 20}`     | `1040`           | Integer |
-| `${@active * 20}`       | `null`           | None    |
-| `${trim(@active)`       | `null`           | None    |
-| `${trim(@consumption)}` | `0.44`           | Float   |
+### object_id
```

Comment:
Any suggestion for this title?

Reply:
Why the current one (object_id) is not appropriated?

Reply:
IMHO, It is not very descriptive if it is not known how expression language works, unlike the other sections.

Reply:
Ok. Understood.

Some alternative title:

> Attribute name duplication in case of multi-entity plugin
> Usage in combination with multi-entity plugin
> Support for `object_id` in multi-entity plugin
> Support for `object_id`

Reply:
Analizan the usage of `object_id`, I realized it is also useful when composing attributes from different measures (`object_id`). For that reason I propose **Attribute value duplication**. ¿What do you think?

Reply:
Rename to context evaluation and, in a subsequent PR add more Information about the context elements

Reply:
Changed the section name here:
https://github.com/telefonicaid/iotagent-node-lib/pull/1158/commits/4e2b6eb680e4a635157b447a95fc781f0fd7a25c